### ISSUE TITLE
Order of headings - Note regions

### ISSUE THREAD

**Issue by giacomo-petri** [w3c/aria-practices#2385]
- **URL**: https://www.w3.org/WAI/ARIA/apg/patterns/dialogmodal/ (or any page containing a NOTE section)
- **Accessibility failure (not following best practices)**: every NOTE section starts with an H5. Even though the NOTE content is ancillary to the specific section and is wrapped within an element with _note_ role, in my opinion it does not fall under the definition of "fixed sections of the page" as it clearly refers to the specific section content (so consistency between pages is not relevant here); it should follow best practices provided in WAI tutorial (https://www.w3.org/WAI/tutorials/page-structure/headings/), in particular:
> Skipping heading ranks can be confusing and should be avoided where possible: Make sure that a \<h2> is not followed directly by an \<h4>, for example. It is ok to skip ranks when closing subsections, for instance, a \<h2> beginning a new section, can follow an \<h4> as it closes the previous section.
- **Expected result**: heading levels follow the meaning of the page structure; in this scenario it should be a heading level 3.
- **Reference**: [W3C - Headings (in Page Structure Tutorial)](https://www.w3.org/WAI/tutorials/page-structure/headings/)
- **Issue snapshot**: <img width="1768" alt="screenshot highlighting poor headings hierarchy structure" src="https://user-images.githubusercontent.com/106394951/174057866-5692a8c1-9413-44dc-857e-2498c4483b23.png">

Giacomo Petri

**Comment by mcking65** [comment]
This has been resolved in production; I checked several pages. I believe the fix was in #2417.
